install/kubernetes/templates/istio-ca-one-namespace.yaml.tmpl

```diff
@@ -34,4 +34,4 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: metadata.namespace
----
+---
```

Comment:
make sure all files end properly with a newline (side note we need a linter for that, I keep repeating it in PR after PR)

Reply:
why is that? it actually creates ugly resulting concatenated files.

Reply:
no it's the only way to concatenate 2 files properly actually
if you have file A
```
line1
line2
```
and file B
```
line3
```
if you cat A B and A doesn't have the last newline you'll get
```
line1
line2line3
```
which is usually not what you want

it's also a sane convention for a files of X lines to have exactly X newlines, not X-1